.release/linux/package/usr/lib/systemd/system/nomad.service

```diff
@@ -26,6 +26,12 @@ After=network-online.target
 # StartLimitInterval = 10s
 
 [Service]
+
+# Nomad clients need to be run as "root" whereas Nomad servers should be run as
+# the "nomad" user. Please change this if needed.
+User=root
+Group=root
+
```

Comment:
It is not clear: is this `nomad.service` file used for nomad server or nomad client?

Reply:
Nomad provides a single Linux package. There is no differentiation at install time (apt/yum/dnf...) between server & client.

Reply:
So users should tune this file to `nomad` user during EC2 instance/DO droplet provisioning for Nomad server.
Do I understand right?

Reply:
Yes (or overwrite it) as part of your configuration management step to setup Nomad and other services on the machine.